.evergreen/atlas/setup-atlas-cluster.sh

```diff
@@ -36,7 +36,8 @@ for VARNAME in ${VARLIST[*]}; do
 done
 
 # Set up the common variables.
-DIR="$(dirname "${BASH_SOURCE[0]}")"
+DIR=$(dirname ${BASH_SOURCE:-$0})
```

Comment:
I actually had to go and double-check why this works. For those who also didn't know, bash allows one to reference array element `0` of an array using scalar notation. For more details, consider the first "Optional Reading" section in [this StackOverflow answer](https://stackoverflow.com/a/35006505).

Reply:
I actually had used that answer.  :).  I added a link.